subprojects/reporting/src/main/java/org/gradle/api/reporting/Report.java

```diff
@@ -71,7 +83,7 @@ public String determineName(Report report) {
      *
      * @return Whether or not this report should be generated by whatever generates it.
      */
-    @Input
+    @ReplacedBy("activated")
```

Comment:
❓ Shouldn't this property and `destination` below be `@Deprecated`?

Reply:
Yes, once the replacements are not `@Incubating` they can be deprecated.